jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/client/ClientUpgradeRequest.java

```diff
@@ -250,6 +261,8 @@ public void onComplete(Result result)
     protected void handleException(Throwable failure)
     {
         futureCoreSession.completeExceptionally(failure);
+        if (frameHandler != null)
+            frameHandler.onError(failure, Callback.NOOP);
```

Comment:
what if `onError` throws here, should this be in a try catch block or it does not matter